openbas-front/src/admin/components/Dashboard.tsx

```diff
@@ -222,6 +226,7 @@ const Dashboard = () => {
             exercises={exercises}
             searchPaginationInput={searchPaginationInput}
             setSearchPaginationInput={setSearchPaginationInput}
+            onOperationSuccess={refreshExercises}
```

Comment:
Can you explain why we need this new mechanism ?
I don't understand

Reply:
When you add or delete a simulation from the Dashboard, the list is not updated. This new mechanism allows me to propagate the change to the parent and forces it to update the list

Reply:
Can you show me where it's possible to add or delete a simulation from the Dashboard screen ?